tests/mqtt-sub-rules/test.yaml

```diff
@@ -109,3 +119,9 @@ checks:
       match:
         event_type: alert
         alert.signature: MQTT SUBSCRIBE topicY
+
+  - filter:
```

Comment:
This one I think we'll have to add a `min-version` check for 8, before this can be merged.

Reply:
Oh, I didn't know you could also have `min-version` on individual filters. Sure, makes sense.